README.md

```diff
@@ -134,6 +134,13 @@ See the [Codegen config](#codegen-config) subsection for the [NextJS](#nextjs) t
 npx projen new --from @ottofeller/templates ottofeller-cdk
 ```
 
+### SST
+An [SST](https://sst.dev) based template for easy deployment of modern full-stack applications on AWS.
+
+```sh
+npx projen new --from @ottofeller/templates ottofeller-sst
```

Comment:
@quesabe are there any options specific to SST? Can we mention them in the docs?

Reply:
Initially I've done it similar to the `cdk` template doc - very short. But yes, there is a single unique option and a few common ones. That's great to have them listed, thus I've the section. Probably a similar update would be great for `cdk`.